films/tests.py

```diff
@@ -3,45 +3,40 @@
 from .models import Actor, Film, Director
 
 class ActorTestCase(TestCase):
-    def setup(self):
+    def setUp(self):
         Actor.objects.create(name='Saoirse', last_name='Ronan', birthdate=date(1994 , 4 , 12), country='USA')
         Actor.objects.create(name='Rami', last_name='Malek', birthdate=date(1981 , 5 , 12), country='USA')
     
     def test_get_all_actors(self):
-        self.setup()
         total_actors = Actor.objects.all()
         self.assertEqual(len(total_actors), 2)
 
     def test_get_actor_age(self):
```

Comment:
You should mock `datetime.date.today`
Like here: https://stackoverrun.com/es/q/5162476#45957644